src/ResourceManager/Network/Commands.Network/Models/PSUsageCustom.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
```

Comment:
Handwritten code as it is customization. Wouldn't be overwritten by generated code